src/main/java/org/example/Main.java

```diff
@@ -37,4 +47,12 @@ public static void main(String[] args) {
             }break;
      }
     }
+    public static void square(double a){
+        int res;
+        res= (int) (a*a);
+        System.out.println("Квадрат: "+res);
+    }
+    public static void root(double a,double b){
+        int res;
```

Comment:
you dont have return here and you are not doing nothing with value int res, fix it